src/python/requirements-app.txt

```diff
@@ -1,2 +1,16 @@
 ### your custom requirements here
-#protobuf==3.13.0
\ No newline at end of file
+black
+flake8
+graphistry[ai]
+ipython
+isort
+jupyter
+mypy
+natsort
+networkx
+numpy
+pytest
+python-dotenv
+splunk-sdk
+streamlit
```

Comment:
????

Reply:
See https://github.com/graphistry/streamlit-avr-app-kit/blob/incident-group-explorer/src/python/requirements-app.txt

These have been pared down all the way to the minimum on my branch of my template repo, unless we are pip freezing?

Reply:
Also remove the others provided by the base image:

```
graphistry[ai]
ipython
numpy
```

Reinstalling risks messing up rapids core deps

Reply:
(if you have some other dockerfile that's not gak you want to support, that can make sense as another file, but shouldn't be what the dockerfile layer builds)

Reply:
you can shell into the base image to see what's there

if helpful:

* most of base image: https://github.com/graphistry/graphistry/blob/master/compose/dockerfiles/base/05-nvidia.Dockerfile

* only faiss-cpu was missing, which I started adding in 2.40.6 @ https://github.com/graphistry/graphistry/pull/1946